src/commands/shop/create.ts

```diff
@@ -0,0 +1,52 @@
+import FluorineClient from '@classes/Client';
+import Embed from '@classes/Embed';
+import { CommandInteraction } from 'discord.js';
+import { ShopItem } from 'types/shop';
+export async function run(
+    client: FluorineClient,
+    interaction: CommandInteraction
+) {
+    const [name, description, price, role] = [
+        interaction.options.getString('name'),
+        interaction.options.getString('description'),
+        interaction.options.getInteger('price'),
+        interaction.options.getRole('role')
+    ];
```

Comment:
```suggestion
    const name = interaction.options.getString('name');
    const description = interaction.options.getString('description');
    const price = interaction.options.getString('price');
    const role =interaction.options.getRole('role');
```
consistency with other declarations and more readable